app/services/pagination.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Pagination
```

Comment:
You can, if you like, end up with this:
```
class Pagination
  class << self
    def page(page_param)
      page_param || '1'
    end

    def set_max_in_page(page, record_count, records_per_page)
      page <= max_page(record_count, records_per_page)
    end

    def max_page(count, records_per_page)
      (count / records_per_page.to_f).ceil
    end
  end
end
```

Reply:
Done! :) Thanks for elaborating on Slack